### ISSUE TITLE
🐛 BUG: Unable to call Astro.fetchContent twice in a component

### ISSUE THREAD

**Issue by lindsaykwardell** [snowpackjs/astro#1496]
### What package manager are you using?

npm

### What operating system are you using?

MacOS BigSur

### Describe the Bug

When I try to call `Astro.fetchContent` more than once in a component, I get an error. Depending on the order of the statements, I'm getting different errors, but either way it appears that I cannot invoke fetchContent more than once in a component. Is this intentional?

Example code:
```js
// index.astro

const blogPosts = Astro.fetchContent('./blog/*.md');
const otherContent = Astro.fetchContent('../content/*.md');
```

The above results in this message:
```
SyntaxError: [62:14]: Missing initializer in const declaration
```

If I reverse those statements (so that `otherContent` is declared above `blogPosts`), I get a different error:
```
SyntaxError: [75:17]: Unexpected token: '{'
```

If I perform the same call twice (two invocations to get the same content, I get yet another error:

```
Error: Parse error @:132:1
```

### Steps to Reproduce

1. `npm init astro` using template blog
2. On the `index.astro`, duplicate the call to `Astro.fetchContent`.
3. Error!

Expected the fetchContent to successfully run twice.

**Comment by matthewp** [comment]
This is working in `next`: https://stackblitz.com/edit/github-qkenhj?file=src/pages/index.astro

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
